src/cf-buttons/src/atoms/buttons.less

```diff
@@ -169,4 +170,14 @@
       padding-bottom: unit( @btn-v-padding-modifier-ie * 15px / @btn__super-font-size, em );
     }
   }
+
+  //
+  // Full width button on x-small screens
+  //
+  &__full-on-xsmall {
```

Comment:
Should we call this `full-on-xs` to match the shorthand used in the breakpoint variables?